javaparser-testing/src/test/java/com/github/javaparser/printer/lexicalpreservation/transformations/ast/body/AnnotationMemberDeclarationTransformationsTest.java

```diff
@@ -50,7 +49,7 @@ protected AnnotationMemberDeclaration consider(String code) {
     // Name
 
     @Test
-    public void changingName() throws IOException {
+    public void changingName() {
```

Comment:
Thanks for removing the useless throws declarations

Reply:
Oh, I just hit "clean file" or something in Intellij ;-)